hiplot/requirements.txt

```diff
@@ -0,0 +1,5 @@
+hiplot
+jupyter
+sklearn
+pandas  # used in hiplot/plot_study.ipynb
```

Comment:
Can I remove this comment because this directory includes only one executable file?
```suggestion
pandas
```

Reply:
I think we can remove the comment.